### PR TITLE
feature: Add more connect options

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description

This will add:
- Social login with Torus
- Seperate wallet-connect button for GoodDollar wallet

About # (link your issue here)
#280 
#272 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

**Comment by L03TJ3** [comment]
UI:
image

**Comment by L03TJ3** [comment]
The current status is that there is a general bug with onboard/torus when switching chains (also happens on their demo/default package)
I am on discord waiting for a response, it likely will become a GitHub issue on their repo

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

File: src/connectors/modules/Torus/torus.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import type { WalletInit } from '@web3-onboard/common'
+import type { TorusCtorArgs, TorusParams } from '@toruslabs/torus-embed'
+
+type TorusOptions = TorusCtorArgs & TorusParams
+
+function torus(options?: TorusOptions): WalletInit {
```
[the remark this answers is not in the document]
Comment (L03TJ3) [reply src/connectors/modules/Torus/torus.ts:6]: This is just done to be able to change the Label of the button, nothing more

File: src/connectors/modules/CustomWc/EthProvider.ts [hunk 2]
```diff
@@ -0,0 +1,204 @@
+import WalletConnect from '@walletconnect/client'
```
[the remark this answers is not in the document]
Comment (L03TJ3) [reply src/connectors/modules/CustomWc/EthProvider.ts]: PR created: https://github.com/GoodDollar/GoodWeb3-Mono/pull/39
draft implementation with last commit here

**Review by johnsmith-gooddollar — APPROVED** [review]

**Comment by L03TJ3** [comment]
@johnsmith-gooddollar the answer to that comment is that this is/was legacy code, and after looking at it I decided that it was not necessary to keep
update fetchListUpdate to use onboard library, tested, didn't see any problems arise from this change

**Review by johnsmith-gooddollar — APPROVED** [review]

**Pull request merged**